st2client/st2client/commands/auth.py

```diff
@@ -70,6 +73,84 @@ def run_and_print(self, args, **kwargs):
                               attributes=self.display_attributes, json=args.json, yaml=args.yaml)
 
 
+class LoginCommand(resource.ResourceCommand):
+    display_attributes = ['user', 'token', 'expiry']
+
+    def __init__(self, resource, *args, **kwargs):
+
+        kwargs['has_token_opt'] = False
+
+        super(LoginCommand, self).__init__(
+            resource, kwargs.pop('name', 'create'),
+            'Authenticate user, acquire access token, and update CLI config directory',
+            *args, **kwargs)
+
+        self.parser.add_argument('username',
+                                 help='Name of the user to authenticate.')
+
+        self.parser.add_argument('-p', '--password', dest='password',
+                                 help='Password for the user. If password is not provided, '
+                                      'it will be prompted.')
+        self.parser.add_argument('-l', '--ttl', type=int, dest='ttl', default=None,
+                                 help='The life span of the token in seconds. '
+                                      'Max TTL configured by the admin supersedes this.')
+        self.parser.add_argument('-w', '--write-real-password', action='store_true', default=False,
```

Comment:
Additionally, don't forget to update Pull Request description (first message) with new syntax/logic, since we don't save fake password: `notarealpassword` anymore